src/app/item-page/simple/item-types/publication/publication.component.html

```diff
@@ -100,6 +100,15 @@
       [fields]="['datacite.relation.isReferencedBy']"
       [label]="'item.page.referenced'">
     </ds-item-page-uri-field>
+    <!-- Below is an example of how to render one or more lat/lng points and/or bounding box rectangles
+         in a tiled map viewer. Set 'cluster' to true for marker clustering -->
+<!--    <ds-geospatial-item-page-field [item]="object"-->
+<!--                                   [label]="'item.page.places'"-->
+<!--                                   [pointFields]="['dcterms.spatial']"-->
+<!--                                   [bboxFields]="['external.spatial.bbox']"-->
+<!--                                   [cluster]="true"-->
+<!--    >-->
+<!--    </ds-geospatial-item-page-field>-->
```

Comment:
Should this commented out code be removed?  Or are you supposed to comment it out in order to use it?

If the expectation is you have to uncomment it, then I'd prefer if we had a way to enable this dynamically via a configuration in your `config.yml`.  That would allow you to more easily turn this feature "on" or "off" in the same way that we do for other features on the Item pages (e.g. mediaviewer, iiif, etc)

Reply:
Keep in mind, if there's no way to "enable" this via configuration then it won't be enabled on our Demo site (demo.dspace.org) and will therefore be *difficult* to support / debug bug tickets/issues (as developers will have to manually enable it to test that the component is working properly). 

It also means that we can never create e2e tests related to this feature, as we cannot modify the HTML in an e2e test.  While we don't require e2e tests at this time, there is a goal to build more of them in #3978

I still find this approach problematic for those reasons.  But, that said, I won't block this PR because of this approach.  It just seems like it's not easy to setup.